src/modules/esl-footnotes/core/esl-note.ts

```diff
@@ -83,14 +83,21 @@ export class ESLNote extends ESLBaseElement {
     return this.allowFootnotes ? `${this._index}` : this.standaloneLabel;
   }
 
+  /** Note settings of ignoring by footnotes */
+  protected get ignoreFootnotes(): string {
+    return this._ignoreFootnotes;
```

Comment:
Why is it in the past tense?

Reply:
I meant using adjective + noun, like this
```suggestion
  protected get ignoredFootnotes(): string {
    return this._ignoredFootnotes;
```

Reply:
Hm... I see the points from both sides and even use some more names with the verb 'ignore' in the comment bellow.
But to be sure I agree with @yadamskaya in the way that it might sound strange, especially out of the context.
But I'm just wondering how can we rephrase it... Just the past tense sounds clear, but it sounds wrong as it has a little bit different meaning...

Reply:
This is exactly the point that sounds bad and is designed crookedly.

Thanks, everyone!